content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,183 @@
+---
+title: April '21 Community Gems
+date: 2021-04-30
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running CML pipeline reports and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running CML pipeline reports  and more.
+picture: 2021-04-30/gems-cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-community-gems/
+tags:
+  - Discord
+  - Gems
+  - DVC
+  - CML
+  - Remote Storage
+---
+
+Each month we go through our Discord messages to pull out some of the best
+questions from our community. AKA: Community Gems. 💎 This month we'd like to
+thank asraniel, Julian, mattlbeck, Ahti, **\_**, yikeqicn, lexzen, EdAb,
+FreshLettuce for inspiring this month's gems!
+
+As always, [join us in Discord](https://discord.com/invite/dvwXA2N) to get all
+your DVC and CML questions answered!
+
+## DVC
+
+### [Can directories and subdirectories be created on OwnCloud with a WebDav remote when I run dvc push if they don't already exist?](https://discord.com/channels/485586884165107732/485596304961962003/831472645508694046)
+
+This likely means that there is an issue with either the credentials or
+permissions set up in WebDav. Make sure that you are using the correct protocol
+and provide a username and/or token or password.
+
+If you use webdavs for your URL and use the `ask_password` parameter for your
+config, this should clear up the issue.
+
+### [What procedures would I use if I needed to remove unwanted or outdated experiments and corresponding data in the cache?](https://discord.com/channels/485586884165107732/485596304961962003/831127462544146482)
```

Comment:
Is it a correct link? Sorry, I'm lost here - is it about `exp` command or more about commits?